web/src/main/java/com/bazaarvoice/emodb/web/auth/Permissions.java

```diff
@@ -73,6 +83,17 @@
                             Conditions.intrinsic(Intrinsic.TABLE, Conditions.like("__*")),
                             Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.like("*:sys")))));
 
+    /**
+     * NON_SYSTEM_TABLE along with excluding PII table/placements.
+     */
+    public final static ConditionResource NON_SYSTEM_NON_PII_TABLE = new ConditionResource(
+            Conditions.not(
+                    Conditions.or(
+                            Conditions.intrinsic(Intrinsic.TABLE, Conditions.like("__*")),
+                            Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.like("*:sys")),
+                            Conditions.intrinsic(Intrinsic.TABLE, Conditions.like("pii:*")),
+                            Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.like("pii_*")))));
```

Comment:
I think it's more accurate to use `Conditions.like("*:pii")` instead of checking for "pii_" as the prefix.  Granted both work with our placement names but the convention of "_x_:_y_" placement names, where _x_ is the Cassandra keyspace and _y_ generally describes the data type (which then maps to a delta table) it seems more natural to use the ":pii" constant suffix.